docs/vault.md

```diff
@@ -8,39 +8,28 @@ builder assumes a logged-in Vault client, probably running on the master server.
 
 Run the following command to log in:
 
-```
-# from projects/elife.yaml, `defaults.aws.vault`
-VAULT_ADDR=https://master-server.elifesciences.org:8200 vault login
-```
+`./bldr vault.login`
 
 You should be asked for an access token that your system administrators should provide you with.
 
 In case it's necessary to log out, run:
 
-```
-rm ~/.vault-token
-```
+`./bldr vault.logout`
 
-Terraform, wrapped by builder, will read the ~/.vault-token file during its operations. There is no need to keep providing the `VAULT_ADDR` environment variable after login, as builder contains the address in its configuration.
+Terraform, wrapped by builder, will read the ~/.vault-token file during its operations.
 
 ## Creating new tokens
 
 After logging in with a root token, run:
 
-```
-VAULT_ADDR=<ADDRESS> vault token create -policy=builder-user -display-name=NameSurname
-```
+`./bldr vault.create_token`
```

Comment:
Being this a layer of indirection upon the binary `vault`, I'd stick to the convention it has introduced: `vault.token.create` (or `vault.token_create` if that's not possible)